build.gradle.kts

```
@@ -28,6 +29,23 @@ detekt {
 
 }
 
+koverReport {
+    val excludedFiles = listOf(
+        "*.BuildConfig.*",
+        "*.BuildConfig",
+        // Jetpack Compose
+        "*.ComposableSingletons*",
+        "*.*\$*Preview\$*",
+        "*.ui.preview.*",
```

According to my experiences, we need to exclude these things:
```suggestion
        "io.mockative.*",
        "*.BuildConfig",
        "*.BuildKonfig",                        // BuildKonfig generated
        "*.ComposableSingletons*",              // Jetpack Compose generated
        "*.*\$*Preview\$*",                     // Jetpack Compose Preview functions
        "*.di.*",                               // Koin
        "*.ui.preview.*",                       // Jetpack Compose Preview providers
        "*.*Test",                              // Test files
        "*.*Test*",                             // Test cases
        "*.*Mock",                              // mockative @Mock generated
        "*.test.*",                             // Test util package
        "*.*\$\$serializer",                    // Kotlinx serializer
```